go-controller/pkg/metrics/master.go

```diff
@@ -68,6 +68,38 @@ var MetricResourceUpdateLatency = prometheus.NewHistogramVec(prometheus.Histogra
 	[]string{"name"},
 )
 
+// MetricRequeueServiceCount is the number of times a particular service has been requeued.
+var MetricRequeueServiceCount = prometheus.NewCounterVec(prometheus.CounterOpts{
+	Namespace: MetricOvnkubeNamespace,
+	Subsystem: MetricOvnkubeSubsystemMaster,
+	Name:      "requeue_service_total",
+	Help:      "A metric that captures the number of times a service is synced with OVN load balancers"},
```

Comment:
C&P on the help text from sync_service_total?

Reply:
uff, thanks